src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/PersonController.kt

```diff
@@ -5,17 +5,18 @@ import org.springframework.security.core.context.SecurityContextHolder
 import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.PersonApiDelegate
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Person
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.PersonRisks
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.config.AuthAwareAuthenticationToken
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.AuthorisableActionResult
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.ForbiddenProblem
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.NotFoundProblem
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.OffenderService
-import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.PersonTransformer
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.RisksTransformer
 
 @Service
 class PersonController(
   private val offenderService: OffenderService,
-  private val personTransformer: PersonTransformer
```

Comment:
This might make more sense to get rid of in 96cc01b18f052a2cadfafc9f996a98de87fe5a61, where it no longer gets used. No biggie, just caused a bit of misdirection for me.

Reply:
Ah this is actually a mistake on my part - should have used the transformer - have rebased in